pkg/ovs/openflow/interfaces.go

```diff
@@ -146,6 +146,36 @@ type Table interface {
 	GetNext() uint8
 	SetNext(next uint8)
 	SetMissAction(action MissActionType)
+	GetStageID() StageID
+}
+
+type StageID uint8
+
+const (
+	Shared StageID = iota
```

Comment:
I don't understand what `Shared` stands for here, same question for `pipelineProtocol`

Reply:
`Shared` means a reserved one. In flexible pipeline, table 0 is special. It used to distribute traffic to different pipelines.
So when creating table 0, its `pipelineProtocol` is special, I thought of using a reserved `pipelineProtocol` named `shared`. So does `StageID`. But I think just using `0` is also ok.